api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_eager_iter.py

```diff
@@ -0,0 +1,92 @@
+import threading
+from collections import deque
+from concurrent import futures
+from typing import Deque, Iterator, Optional, TypeVar
+
+_T = TypeVar("_T")
+
+
+class EagerIterator(Iterator[_T]):
```

Comment:
This could be useful to have in a sibling package, since it's useful elsewhere and not preprocessing-specific. E.g., I suspect it can be leveraged by `pytorch.py`, I can move when it's incorporated elsewhere.

Reply:
feel free to grab/move it whenever you find a second use case. I didn't want to presume it would be useful until it was :-)